MediaBrowser.Providers/Manager/MetadataService.cs

```diff
@@ -252,18 +252,15 @@ private void SavePeopleMetadata(List<PersonInfo> people, LibraryOptions libraryO
 
         private void AddPersonImage(Person personEntity, LibraryOptions libraryOptions, string imageUrl, CancellationToken cancellationToken)
         {
-            //if (libraryOptions.DownloadImagesInAdvance)
-            //{
-            //    try
-            //    {
-            //        await ProviderManager.SaveImage(personEntity, imageUrl, ImageType.Primary, null, cancellationToken).ConfigureAwait(false);
-            //        return;
-            //    }
-            //    catch (Exception ex)
-            //    {
-            //        Logger.LogError(ex, "Error in AddPersonImage");
-            //    }
-            //}
+            try
```

Comment:
Not sure why this was commented. Any clue?

Reply:
Probably because of the return? Which make the SetImage code redundant and only execute when there is an error?

Reply:
I know I'm a bit late to the party, but it was probably commented because it makes scanning hilariously slow. @Bond-009 enabled it and killed scanning speed 😅

Reply:
Did we disable it again?